docs/usecases/RStudio_user.rst

```diff
@@ -0,0 +1,228 @@
+`create a new project` function using the SSH address of the parent repository.
+
+
+.. figure:: ../artwork/src/rstudio/Rstudio-create.png
+   :scale: 80 %
+   :alt: screenshots patchwork of Rstudio new project creation.
+
+   Figures of several screenshots demonstrating the creation of new projects on Rstudio.
+
+Max can't see submodules content and comes to Bobby.
+
+Bobby comes and run ``datalad get . -n -r`` in the terminal window of Rstudio. 
+
+.. figure:: ../artwork/src/rstudio/Rstudio-terminal.png
+   :scale: 80 %
+   :alt: screenshot of Rstudio terminal tab.
+
+   Using Rstudio terminal window to give DataLad commands
+
+
+They then explain:
+  - Rstudio can only use simple Git commands, which do not clone submodule content.
+  - DataLad command are run in the terminal window. DataLad does not have a R package and do not run in the console.
+  - This specific function :command:`datalad get .` will download all files, it has two options:
+     - ``-n`` option means annexed files will not be downloaded
+     - ``-r`` option (short for ``--recursive``) means that the function is run in all submodules, recursively.
+
+Max is thanking Bobby for the insights.
+Before leaving, Bobby gives an additional advice: Our template uses "pure Git" repositories, DataLad functions will work but they will not use git-annex. 
+Looking at Max incredulous stare, they explain further: you will now be able to use DataLad to manage the submodules and save them all at once, but big files will be added to Git, this will make it unusable very fast. 
+So you need to turn this pure Git repository into proper :term:`DataLad dataset` (meaning a Git repository with additional features from :term:`git-annex` and DataLad).
+
+Max is a bit puzzled and read the basics chapter of the handbook again.
+Then, they see that :command:`datalad create --force` is the correct command  to create a DataLad dataset.
+when the folder already exist, so they  run 
+``datalad create --force -r`` in the parent repository.
+Now they are sure they set up DataLad to work in the repository and all submodules,
+since they used the ``-r``  option.
+Max also now understand what Bobby meant when he said that annexed files are not downloaded, and this is further explained in the next section.
+
+.. gitusernote:: Pushing and GIN-Tonic
+
+  The scenario  does not present the use of remotes (server version of the resitory) and :command:`datalad push`, as it is indeed not different for R or Rstudio users.
+  One *only* needs to use DataLad commands (in the terminal tab) instead of Git (integration in Rstudio-GUI or shell).
+  
+  While the original repositories are described as pure Git repositories here,
+  most of the scenario was actually tested
+  using a :term:`GIN` repository with git-annex, with :term:`submodule`\s 
+  for data and for code, that were created using the `Tonic tool and templates <https://gin-tonic.netlify.app>`_ . 
```

Comment:
```suggestion
  for data and for code, that were created using the `Tonic tool and templates <https://gin-tonic.netlify.app>`_ , `see archived version <https://doi.org/10.5281/zenodo.7763694>`_
```

Reply:
adding a link, so that if ever the website is put down, there is still a working link.